server/services/licenceService.ts

```diff
@@ -198,6 +201,39 @@ export default class LicenceService {
     return this.licenceApiClient.updateAdditionalConditionData(licenceId, additionalConditionId, requestBody, user)
   }
 
+  async uploadExclusionZoneFile(
+    licenceId: string,
+    additionalConditionId: string,
+    fileToUpload: Express.Multer.File,
+    user: User,
+    testMode = false
+  ): Promise<void> {
+    logger.info(`Called uploadExclusionZoneFile - name ${fileToUpload.originalname} path ${fileToUpload.path}`)
+    await this.licenceApiClient.uploadExclusionZoneFile(licenceId, additionalConditionId, user, fileToUpload)
+    if (!testMode) {
+      logger.info(`Removing file ${fileToUpload.path} after uploading content to API`)
+      fs.unlinkSync(fileToUpload.path)
```

Comment:
Does the file uploaded by the user get stored on the node file system server side?

Reply:
It is temporarily stored on the node file system and removed after uploading to the API. I would prefer not to do this, but struggled for alternatives.